File: dataset/Vanilla/GEMINI_gemini-2.5-pro/Task197.py
import random
import math
from typing import List

class Solution:

    def __init__(self, radius: float, x_center: float, y_center: float):
        self.radius = radius
        self.x_center = x_center
        self.y_center = y_center

    def randPoint(self) -> List[float]:
        angle = 2 * math.pi * random.random()
        r = self.radius * math.sqrt(random.random())
        x = self.x_center + r * math.cos(angle)
        y = self.y_center + r * math.sin(angle)
        return [x, y]

if __name__ == '__main__':
    s1 = Solution(1.0, 0.0, 0.0)
    print(s1.randPoint())
    print(s1.randPoint())
    print(s1.randPoint())

    s2 = Solution(10.0, 5.0, -7.5)
    print(s2.randPoint())
    print(s2.randPoint())

    s3 = Solution(0.01, -100.0, 100.0)
    print(s3.randPoint())

    s4 = Solution(1000.0, 0.0, 0.0)
    print(s4.randPoint())

    s5 = Solution(2.0, 2.0, 2.0)
    print(s5.randPoint())

File: dataset/Vanilla/GEMINI_gemini-2.5-pro/Task197.java
import java.util.Random;
import java.util.Arrays;

class Solution {
    private double radius;
    private double x_center;
    private double y_center;
    private Random rand;

    public Solution(double radius, double x_center, double y_center) {
        this.radius = radius;
        this.x_center = x_center;
        this.y_center = y_center;
        this.rand = new Random();
    }
    
    public double[] randPoint() {
        double angle = 2 * Math.PI * rand.nextDouble();
        double r = this.radius * Math.sqrt(rand.nextDouble());
        double x = this.x_center + r * Math.cos(angle);
        double y = this.y_center + r * Math.sin(angle);
        return new double[]{x, y};
    }
}

public class Task197 {
    public static void main(String[] args) {
        Solution s1 = new Solution(1.0, 0.0, 0.0);
        System.out.println(Arrays.toString(s1.randPoint()));
        System.out.println(Arrays.toString(s1.randPoint()));
        System.out.println(Arrays.toString(s1.randPoint()));
        
        Solution s2 = new Solution(10.0, 5.0, -7.5);
        System.out.println(Arrays.toString(s2.randPoint()));
        System.out.println(Arrays.toString(s2.randPoint()));
        
        Solution s3 = new Solution(0.01, -100.0, 100.0);
        System.out.println(Arrays.toString(s3.randPoint()));
        
        Solution s4 = new Solution(1000.0, 0.0, 0.0);
        System.out.println(Arrays.toString(s4.randPoint()));
        
        Solution s5 = new Solution(2.0, 2.0, 2.0);
        System.out.println(Arrays.toString(s5.randPoint()));
    }
}